R/cache.R

```diff
@@ -532,8 +532,10 @@ data_injector_cached <- function(fcn_call, keys) {
 read_df_from_a_shard <- function(fcn_call, keys, shard) {
   sql <- paste("SELECT * FROM", shard, "WHERE", fcn_call$output_key, "IN (",
                paste(sanitize_sql(keys), collapse = ', '), ")")
-  db2df(dbGetQuery(fcn_call$con, sql),
+  df <- db2df(dbGetQuery(fcn_call$con, sql),
         fcn_call$con, fcn_call$output_key)
+  df$last_cached_at <- NULL
```

Comment:
Hmm, we might have to do it earlier... *before* we translate columns back in `db2df`. Otherwise will drop `last_cached_at` from people's data if it's a legitimate column (even if it's MD5 hashed in the table).